src/state/slices/txHistorySlice/selectors.ts

```diff
@@ -100,6 +100,14 @@ const selectWalletTxsByAccountIdAssetId = createSelector(
     pickBy(txsByAccountIdAssetId, (_, accountId) => accountIds.includes(accountId)),
 )
 
+export const selectArbitrumWithdrawTxs = createSelector(selectTxs, (txs): Tx[] => {
```

Comment:
explicit selector for arbitrum withdraw txs to easily use type narrowing on parser.